### ISSUE TITLE
GitHub API error

### ISSUE THREAD

**Issue by vazquez** [mozilla/build.webmaker.org#81]
When running `gulp liveserve` I receive this error:

![screenshot 2015-07-28 10 03 55](https://cloud.githubusercontent.com/assets/1119821/8933336/36752a62-3510-11e5-8f5c-735a0622d1f3.png)

Does this have something to do with the latest commits you pushed up?

**Comment by ScottDowne** [comment]
Interesting... Could it be missing github credentials in the .env file?

**Comment by ScottDowne** [comment]
Try npm start

**Comment by ScottDowne** [comment]
Yeah, so no idea what gulp liveserve is doing, doesn't work for me, but npm start does for me.

That said, if I remove my github credentials from my .env I get the same error you see, so that's probably it.

I'll add a bit of better handling for this.

**Comment by ScottDowne** [comment]
This should make it not fail without github creds.

https://github.com/mozilla/build.webmaker.org/pull/82

**Comment by vazquez** [comment]
Still receive an error when running `npm start` :/

<img width="1198" alt="screenshot 2015-07-28 11 01 58" src="https://cloud.githubusercontent.com/assets/1119821/8934934/22e6d470-3518-11e5-8fe9-c62a433d7a10.png">

**Comment by vazquez** [comment]
Ah cool :)

**Comment by vazquez** [comment]
I was able to fix the problem by removing the following line from the `.env` file

`export PLAN_GITHUB_TOKEN="xxx"`
